docs/token-api.md

```diff
@@ -109,23 +125,91 @@ It consists of three parts:
     - `Transfers`, to manipulate Tokens (e.g., transfer ownership or redeem)
 
   The actions in the collection are independent. An action cannot spend tokens created by another action in the same Token Request.
-  In addition, actions comes with a set of `Witnesses` to verify the `right to spend` or the `right to issue` a given token
+  In addition, actions comes with a set of `Witnesses` to verify the `right to spend` or the `right to issue` a given token.
+  In the simplest case, the witnesses are the signatures of the issuers and the token owners.
```

Comment:
similar to what an endorsement is in Fabric ... ?!

Reply:
I would say no.